statistics/src/main/java/com/duckduckgo/app/statistics/pixels/Pixel.kt

```diff
@@ -18,22 +18,25 @@ package com.duckduckgo.app.statistics.pixels
 
 import android.annotation.SuppressLint
 import com.duckduckgo.app.statistics.api.PixelSender
-import com.duckduckgo.app.statistics.pixels.Pixel.PixelName
 import io.reactivex.schedulers.Schedulers
 import timber.log.Timber
 import javax.inject.Inject
 
 interface Pixel {
 
-    enum class PixelName(val pixelName: String) {
+    interface PixelName {
+        val pixelName: String
+    }
+
+    enum class StatisticsPixelName(override val pixelName: String) : PixelName {
+        WEB_RENDERER_GONE_CRASH("m_d_wrg_c"),
+        WEB_RENDERER_GONE_KILLED("m_d_wrg_k"),
 
-        APP_LAUNCH("ml"),
 
-        FORGET_ALL_PRESSED_BROWSING("mf_bp"),
-        FORGET_ALL_PRESSED_TABSWITCHING("mf_tp"),
-        FORGET_ALL_EXECUTED("mf"),
-        FORGET_ALL_AUTO_RESTART("m_f_r"),
-        FORGET_ALL_AUTO_RESTART_WITH_INTENT("m_f_ri"),
+        COOKIE_DATABASE_NOT_FOUND("m_cdb_nf"),
+        COOKIE_DATABASE_OPEN_ERROR("m_cdb_oe"),
+        COOKIE_DATABASE_DELETE_ERROR("m_cdb_de"),
+        COOKIE_DATABASE_CORRUPTED_ERROR("m_cdb_ce"),
```

Comment:
These pixels are use in the `OfflinePixelSender` that lives in the `statistics` gradle module.
I think at some point we should also look into not having these pixels in the `statistics` gradle module. Ideally it should only contain the public API for the pixels, but none of the pixel names to be sent.
for now I just left these here